components/contribution-flow/index.js

```diff
@@ -825,16 +826,29 @@ class CreateOrderPage extends React.Component {
                 disabled={this.state.submitting || this.state.submitted}
               />
             </Flex>
-            {this.isFixedContribution() ? (
-              <ContributionDetails
-                totalAmount={get(stepDetails, 'totalAmount')}
-                interval={interval}
-                currency={this.getCurrency()}
-                tax={this.state.stepSummary}
-              />
-            ) : (
-              <Box width={[0, null, null, 1 / 5]} />
-            )}
+            <Container>
```

Comment:
That should be solvable here. I'd try to remove all the `<Box width={[0, null, null, 1 / 5]} />` below and put the `width={[0, null, null, 1 / 5]}` on this container - as the parent, it should be responsible for the width of the column.

Also `ContributionDetails` has `minWidth="300px" maxWidth="370px"`, maybe that's something we want to add as well.

Reply:
Thanks! It seems to work fine, I updated the commit.